packages/terra-rpc-provider/lib/TerraRpcProvider.ts

```diff
@@ -135,4 +135,14 @@ export default class TerraRpcProvider extends NodeProvider implements FeeProvide
       }
     }
   }
+
+  async getTaxFees(amount: number, denom = 'uusd'): Promise<any> {
```

Comment:
Denom shouldn't be optional I think, It should be explicit which taxes you want to retrieve